### ISSUE TITLE
With 0.23.5 a tooltip on disabled button won't disappear

### ISSUE THREAD

**Issue by amarant** [react-bootstrap/react-bootstrap#890]
Hi,
With 0.23.5 a tooltip on a disabled button won't disappear, this appears to follow the commit 2ffcf5d
An example usable on [here](http://react-bootstrap.github.io/components.html#tooltips)
`````javascript
const positionerInstance = (
  <ButtonToolbar>
    <OverlayTrigger placement='right' overlay={<Tooltip><strong>Holy guacamole!</strong> Check this info.</Tooltip>}>
      <Button bsStyle='default' disabled>Holy guacamole!</Button>
    </OverlayTrigger>
  </ButtonToolbar>
);

React.render(positionerInstance, mountNode);
````

Thanks.

**Comment by jquense** [comment]
I am pretty sure this is a React bug, with their mouseEnter/Leave cross browser plugin. I will see if there is a reasonable workaround to fix it here, and submit an issue upstream.

**Comment by AlexKVal** [comment]
It seems this bug has been fixed in version `0.23.7`. :sparkle: 
I can't reproduce it anymore on the docs page http://react-bootstrap.github.io/components.html#tooltips

Feel free to reopen if you still have the issue with `v0.23.7`
:cherries:

**Comment by jquense** [comment]
for posterity: https://github.com/facebook/react/issues/4251
